moveit_ros/planning/trajectory_execution_manager/cfg/TrajectoryExecutionDynamicReconfigure.cfg

```diff
@@ -7,5 +7,6 @@ gen.add("execution_duration_monitoring", bool_t, 1, "Monitor the execution durat
 gen.add("allowed_execution_duration_scaling", double_t, 2, "Accept durations that take a little more time than specified", 1.1, 1, 10)
 gen.add("allowed_goal_duration_margin", double_t, 3, "Allow more than the expected execution time before triggering a trajectory cancel (applied after scaling)", 0.5, 0.1, 5)
 gen.add("execution_velocity_scaling", double_t, 4, "Multiplicative factor for execution speed", 1, 0.1, 10)
+gen.add("allowed_start_tolerance", double_t, 5, "Allowed joint-value tolerance for validation of trajectory's start point against current robot state", 1e-3, 0);
```

Comment:
Looking at this again, I believe it is unnecessarily complex to have this parameter in dynamic reconfigure.

- it's one more place where the default value has to be provided
- if anyone wants to add per-joint thresholds later on, the meaning of this parameter becomes "default if none is specified"
  This meaning does not deserved to be changed dynamically imho
- Personally I don't see a reason why anyone would ever change that value during runtime. This is what dynamic reconfigure is for.

Opinions?